### ISSUE TITLE
Ingest server failure

### ISSUE THREAD

**Issue by LuizStSantos** [ffplayout/ffplayout#582]
- OS: Debian 12
- ffplayout version : 0.20.5
- ffmpeg version  5.1.4-0+deb12u1
- are you using the current master of ffplayout? yes

**Error**

When trying to use the RTMP ingest server I have a problem accessing and deleting old segment, and it keeps connecting and disconnecting , software used to connect OBS Studio

```
[2024-04-05 22:21:28.25948] [ INFO] Switch from playlist to live ingest
[2024-04-05 22:21:31.66409] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-341.ts: No such file or directory
[2024-04-05 22:21:33.69217] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-342.ts: No such file or directory
[2024-04-05 22:21:35.65228] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-343.ts: No such file or directory
[2024-04-05 22:21:37.66387] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-344.ts: No such file or directory
[2024-04-05 22:21:39.66935] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-345.ts: No such file or directory
[2024-04-05 22:21:41.67137] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-346.ts: No such file or directory
[2024-04-05 22:21:43.64511] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-347.ts: No such file or directory
[2024-04-05 22:21:45.64302] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-348.ts: No such file or directory
[2024-04-05 22:21:47.68233] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-349.ts: No such file or directory
[2024-04-05 22:21:49.70686] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-350.ts: No such file or directory
[2024-04-05 22:21:51.64598] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-351.ts: No such file or directory
[2024-04-05 22:21:53.63954] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-352.ts: No such file or directory
[2024-04-05 22:21:55.68069] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-353.ts: No such file or directory
[2024-04-05 22:21:57.68813] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-354.ts: No such file or directory
[2024-04-05 22:21:59.65798] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-355.ts: No such file or directory
[2024-04-05 22:22:01.64099] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-356.ts: No such file or directory
[2024-04-05 22:22:03.69772] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-357.ts: No such file or directory
[2024-04-05 22:22:05.68860] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-358.ts: No such file or directory
[2024-04-05 22:22:07.65521] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-359.ts: No such file or directory
[2024-04-05 22:22:09.63200] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-360.ts: No such file or directory
[2024-04-05 22:22:11.68093] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-361.ts: No such file or directory
[2024-04-05 22:22:15.45843] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-362.ts: No such file or directory
[2024-04-05 22:22:15.64902] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-363.ts: No such file or directory
[2024-04-05 22:22:17.63667] [ERROR] [Server] [hls muxer @ 0x55f165cf9c40] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-364.ts: No such file or directory
[2024-04-05 22:22:19.66706] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-365.ts: No such file or directory
[2024-04-05 22:22:21.67642] [ERROR] [Decoder] [hls muxer @ 0x55ef48897700] failed to delete old segment /usr/share/ffplayout/public/live/stream-0-366.ts: No such file or directory
```

**Comment by jb-alvarado** [comment]
When this is happen can you check with `ps aux` which ffmpeg instances are running? It looks like that decoder and ingest are running in parallel, and that would be a problem.

**Comment by JunioCalu** [comment]
It might be useful to monitor the processes using a loop:

    while true; do sleep 0.1s; ps aux | grep -i ffmpeg --color ; done

**Comment by LuizStSantos** [comment]
> When this is happen can you check with which ffmpeg instances are running? It looks like that decoder and ingest are running in parallel, and that would be a problem.`ps aux`

without running

```
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.0 168940 13244 ?        Ss   abr03   0:04 /sbin/init
root           2  0.0  0.0      0     0 ?        S    abr03   0:00 [kthreadd]
root           3  0.0  0.0      0     0 ?        I<   abr03   0:00 [rcu_gp]
root           4  0.0  0.0      0     0 ?        I<   abr03   0:00 [rcu_par_gp]
root           5  0.0  0.0      0     0 ?        I<   abr03   0:00 [slub_flushwq]
root           6  0.0  0.0      0     0 ?        I<   abr03   0:00 [netns]
root           8  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/0:0H-events_highpri]
root          10  0.0  0.0      0     0 ?        I<   abr03   0:00 [mm_percpu_wq]
root          11  0.0  0.0      0     0 ?        I    abr03   0:00 [rcu_tasks_kthread]
root          12  0.0  0.0      0     0 ?        I    abr03   0:00 [rcu_tasks_rude_kthread]
root          13  0.0  0.0      0     0 ?        I    abr03   0:00 [rcu_tasks_trace_kthread]
root          14  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/0]
root          15  0.0  0.0      0     0 ?        I    abr03   0:29 [rcu_preempt]
root          16  0.0  0.0      0     0 ?        S    abr03   0:05 [migration/0]
root          18  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/0]
root          19  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/1]
root          20  0.0  0.0      0     0 ?        S    abr03   0:33 [migration/1]
root          21  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/1]
root          23  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/1:0H-events_highpri]
root          24  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/2]
root          25  0.0  0.0      0     0 ?        S    abr03   0:03 [migration/2]
root          26  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/2]
root          28  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/2:0H-events_highpri]
root          29  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/3]
root          30  0.0  0.0      0     0 ?        S    abr03   0:32 [migration/3]
root          31  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/3]
root          33  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/3:0H-events_highpri]
root          34  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/4]
root          35  0.0  0.0      0     0 ?        S    abr03   0:03 [migration/4]
root          36  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/4]
root          38  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/4:0H-events_highpri]
root          39  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/5]
root          40  0.0  0.0      0     0 ?        S    abr03   0:36 [migration/5]
root          41  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/5]
root          43  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/5:0H-events_highpri]
root          44  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/6]
root          45  0.0  0.0      0     0 ?        S    abr03   0:05 [migration/6]
root          46  0.0  0.0      0     0 ?        S    abr03   0:03 [ksoftirqd/6]
root          48  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/6:0H-events_highpri]
root          49  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/7]
root          50  0.0  0.0      0     0 ?        S    abr03   0:53 [migration/7]
root          51  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/7]
root          53  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/7:0H-events_highpri]
root          54  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/8]
root          55  0.0  0.0      0     0 ?        S    abr03   0:01 [migration/8]
root          56  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/8]
root          58  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/8:0H-events_highpri]
root          59  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/9]
root          60  0.0  0.0      0     0 ?        S    abr03   0:56 [migration/9]
root          61  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/9]
root          63  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/9:0H-events_highpri]
root          64  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/10]
root          65  0.0  0.0      0     0 ?        S    abr03   0:11 [migration/10]
root          66  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/10]
root          68  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/10:0H-events_highpri]
root          69  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/11]
root          70  0.0  0.0      0     0 ?        S    abr03   1:48 [migration/11]
root          71  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/11]
root          73  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/11:0H-events_highpri]
root          74  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/12]
root          75  0.0  0.0      0     0 ?        S    abr03   0:01 [migration/12]
root          76  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/12]
root          78  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/12:0H-events_highpri]
root          79  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/13]
root          80  0.0  0.0      0     0 ?        S    abr03   0:58 [migration/13]
root          81  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/13]
root          83  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/13:0H-events_highpri]
root          84  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/14]
root          85  0.0  0.0      0     0 ?        S    abr03   0:11 [migration/14]
root          86  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/14]
root          88  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/14:0H-events_highpri]
root          89  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/15]
root          90  0.0  0.0      0     0 ?        S    abr03   5:46 [migration/15]
root          91  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/15]
root          93  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/15:0H-events_highpri]
root          94  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/16]
root          95  0.0  0.0      0     0 ?        S    abr03   0:21 [migration/16]
root          96  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/16]
root          97  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/16:0-mm_percpu_wq]
root          98  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/16:0H-events_highpri]
root          99  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/17]
root         100  0.0  0.0      0     0 ?        S    abr03   0:29 [migration/17]
root         101  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/17]
root         103  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/17:0H-events_highpri]
root         104  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/18]
root         105  0.0  0.0      0     0 ?        S    abr03   1:15 [migration/18]
root         106  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/18]
root         107  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/18:0-mm_percpu_wq]
root         108  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/18:0H-events_highpri]
root         109  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/19]
root         110  0.0  0.0      0     0 ?        S    abr03   2:53 [migration/19]
root         111  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/19]
root         113  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/19:0H-events_highpri]
root         128  0.0  0.0      0     0 ?        S    abr03   0:00 [kdevtmpfs]
root         129  0.0  0.0      0     0 ?        I<   abr03   0:00 [inet_frag_wq]
root         130  0.0  0.0      0     0 ?        S    abr03   0:00 [kauditd]
root         133  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/1:2-cgroup_destroy]
root         134  0.0  0.0      0     0 ?        S    abr03   0:00 [khungtaskd]
root         135  0.0  0.0      0     0 ?        S    abr03   0:00 [oom_reaper]
root         136  0.0  0.0      0     0 ?        I<   abr03   0:00 [writeback]
root         137  0.0  0.0      0     0 ?        S    abr03   0:50 [kcompactd0]
root         138  0.0  0.0      0     0 ?        SN   abr03   0:00 [ksmd]
root         141  0.0  0.0      0     0 ?        SN   abr03   0:39 [khugepaged]
root         142  0.0  0.0      0     0 ?        I<   abr03   0:00 [kintegrityd]
root         143  0.0  0.0      0     0 ?        I<   abr03   0:00 [kblockd]
root         144  0.0  0.0      0     0 ?        I<   abr03   0:00 [blkcg_punt_bio]
root         145  0.0  0.0      0     0 ?        I    abr03   0:07 [kworker/3:1-rcu_gp]
root         146  0.0  0.0      0     0 ?        I<   abr03   0:00 [tpm_dev_wq]
root         147  0.0  0.0      0     0 ?        I<   abr03   0:00 [edac-poller]
root         148  0.0  0.0      0     0 ?        I<   abr03   0:00 [devfreq_wq]
root         150  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/3:1H-kblockd]
root         151  0.0  0.0      0     0 ?        S    abr03   0:02 [kswapd0]
root         158  0.0  0.0      0     0 ?        I<   abr03   0:00 [kthrotld]
root         160  0.0  0.0      0     0 ?        I<   abr03   0:00 [acpi_thermal_pm]
root         161  0.0  0.0      0     0 ?        S    abr03   0:05 [hwrng]
root         162  0.0  0.0      0     0 ?        I<   abr03   0:00 [hfi-updates]
root         163  0.0  0.0      0     0 ?        I<   abr03   0:00 [mld]
root         164  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/2:1H-kblockd]
root         165  0.0  0.0      0     0 ?        I<   abr03   0:00 [ipv6_addrconf]
root         170  0.0  0.0      0     0 ?        I<   abr03   0:00 [kstrp]
root         174  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/11:1-mm_percpu_wq]
root         176  0.0  0.0      0     0 ?        I<   abr03   0:00 [zswap-shrink]
root         177  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/u41:0]
root         196  0.0  0.0      0     0 ?        I    abr03   0:01 [kworker/5:1-events]
root         214  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/9:1-events]
root         226  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/12:1H-kblockd]
root         238  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/8:1H-kblockd]
root         239  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/6:1H-kblockd]
root         240  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/4:1H-kblockd]
root         241  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/15:1H-kblockd]
root         243  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/14:1H-kblockd]
root         246  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/17:1H-kblockd]
root         247  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/10:1H-kblockd]
root         253  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/7:1H-kblockd]
root         281  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/11:1H-kblockd]
root         294  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/1:1H-kblockd]
root         295  0.0  0.0      0     0 ?        I<   abr03   0:03 [kworker/9:1H-kblockd]
root         296  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/5:1H-kblockd]
root         297  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/13:1H-kblockd]
root         298  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/0:1H-kblockd]
root         299  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/16:1H-kblockd]
root         300  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/18:1H-kblockd]
root         301  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/19:1H-kblockd]
root         306  0.0  0.0      0     0 ?        I<   abr03   0:00 [ata_sff]
root         307  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_0]
root         308  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_0]
root         309  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_1]
root         310  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_1]
root         311  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_2]
root         312  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_2]
root         313  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_3]
root         314  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_3]
root         315  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_4]
root         316  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_4]
root         317  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_5]
root         318  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_5]
root         319  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_6]
root         320  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_6]
root         321  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_7]
root         322  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_7]
root         324  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/11:2-pm]
root         329  0.0  0.0      0     0 ?        I    abr03   0:02 [kworker/13:1-events]
root         330  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/16:1-mm_percpu_wq]
root         332  0.0  0.0      0     0 ?        I    abr03   0:05 [kworker/19:1-mm_percpu_wq]
root         333  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/18:1-mm_percpu_wq]
root         370  0.0  0.0      0     0 ?        S    abr03   0:13 [jbd2/sda2-8]
root         371  0.0  0.0      0     0 ?        I<   abr03   0:00 [ext4-rsv-conver]
root         410  0.0  0.4 114852 65524 ?        Ss   abr03   2:34 /lib/systemd/systemd-journald
root         443  0.0  0.0  26932  5968 ?        Ss   abr03   0:00 /lib/systemd/systemd-udevd
root         502  0.0  0.0      0     0 ?        S    abr03   0:00 [nv_queue]
root         503  0.0  0.0      0     0 ?        S    abr03   0:00 [nv_queue]
root         524  0.0  0.0      0     0 ?        S    abr03   0:00 [watchdogd]
root         527  0.0  0.0      0     0 ?        S    abr03   0:00 [irq/126-mei_me]
root         528  0.0  0.0      0     0 ?        S    abr03   2:51 [nvidia-modeset/kthread_q]
root         529  0.0  0.0      0     0 ?        S    abr03   0:00 [nvidia-modeset/deferred_close_kthread_q]
root         532  0.0  0.0      0     0 ?        I<   abr03   0:00 [cryptd]
root         623  0.0  0.0      0     0 ?        S    abr03   0:00 [jbd2/sdb-8]
root         624  0.0  0.0      0     0 ?        I<   abr03   0:00 [ext4-rsv-conver]
root         649  0.0  0.0   6608  2576 ?        Ss   abr03   0:00 /usr/sbin/cron -f
message+     652  0.0  0.0   8044  4300 ?        Ss   abr03   0:00 /usr/bin/dbus-daemon --system --address=systemd: --no
root         663  0.0  0.0  49628  7780 ?        Ss   abr03   0:00 /lib/systemd/systemd-logind
nvpd         668  0.0  0.0   6560  2008 ?        Ss   abr03   0:00 /usr/bin/nvidia-persistenced --user nvpd
root         690  0.1  0.0      0     0 ?        S    abr03   7:04 [irq/128-nvidia]
root         691  0.0  0.0      0     0 ?        S    abr03   0:00 [nvidia]
root         692  0.0  0.0      0     0 ?        S    abr03   0:56 [nv_queue]
root         693  0.0  0.0   5872   980 tty1     Ss+  abr03   0:00 /sbin/agetty -o -p -- \u --noclear - linux
root         709  0.0  0.0  15412  9188 ?        Ss   abr03   0:00 sshd: /usr/sbin/sshd -D [listener] 0 of 10-100 startu
root         742  0.0  0.0      0     0 ?        I    abr03   0:02 [kworker/17:2-mm_percpu_wq]
root        1838  0.0  0.0      0     0 ?        S    abr03   0:00 [UVM global queue]
root        1839  0.0  0.0      0     0 ?        S    abr03   0:00 [UVM deferred release queue]
root        1840  0.0  0.0      0     0 ?        S    abr03   0:00 [UVM Tools Event Queue]
root      194903  0.0  0.0      0     0 ?        I    abr03   0:01 [kworker/1:1-mm_percpu_wq]
root      241634  0.0  0.0      0     0 ?        I    abr04   0:02 [kworker/12:0-rcu_par_gp]
root      348475  0.0  0.0      0     0 ?        I    abr04   0:00 [kworker/19:2]
root      348485  0.0  0.0      0     0 ?        I    abr04   0:03 [kworker/10:2-cgroup_destroy]
root      366218  0.0  0.0      0     0 ?        I    abr04   0:00 [kworker/7:0-events]
root      371115  0.0  0.0      0     0 ?        I    abr04   0:03 [kworker/15:2-mm_percpu_wq]
root      375139  0.0  0.0      0     0 ?        I    abr05   0:00 [kworker/13:2-rcu_par_gp]
root      452990  0.0  0.0      0     0 ?        I    abr05   0:07 [kworker/3:2-events]
root      452991  0.0  0.0      0     0 ?        I    abr05   0:07 [kworker/0:1-events]
root      463027  0.0  0.0      0     0 ?        I    abr05   0:00 [kworker/15:0]
ffpu      488824  0.6  0.2 173256 45496 ?        Ssl  abr05  18:25 /usr/bin/ffpapi -l 0.0.0.0:8787
root      579556  0.0  0.0      0     0 ?        I    abr05   0:00 [kworker/17:0]
root      580240  0.0  0.0      0     0 ?        I    abr05   0:01 [kworker/14:1-rcu_par_gp]
root      624324  0.0  0.0      0     0 ?        I    abr06   0:02 [kworker/12:1-events]
root      624381  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/9:0]
root      631378  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/5:0]
root      665516  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/4:0-rcu_gp]
root      665517  0.0  0.0      0     0 ?        I    abr06   0:01 [kworker/2:1-events]
root      693607  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/7:2-cgroup_destroy]
root      693610  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/0:0]
ffpu      723212  0.0  0.0  26740  9172 ?        Ssl  abr06   0:01 /usr/bin/ffplayout
root      723213  0.0  0.0      0     0 ?        I    abr06   0:04 [kworker/6:0-mm_percpu_wq]
root      747056  0.0  0.0      0     0 ?        I    00:00   0:00 [kworker/14:2-mm_percpu_wq]
root      747057  0.0  0.0      0     0 ?        I    00:00   0:01 [kworker/8:0-mm_percpu_wq]
root      748462  0.0  0.0      0     0 ?        I    01:53   0:00 [kworker/2:2]
root      750242  0.0  0.0      0     0 ?        I    03:10   0:00 [kworker/8:1]
root      753225  0.0  0.0      0     0 ?        I    06:16   0:00 [kworker/6:2-cgroup_destroy]
root      787720  0.0  0.0      0     0 ?        I    17:43   0:00 [kworker/4:1-events]
root      787721  0.0  0.0      0     0 ?        I    17:43   0:00 [kworker/10:1-mm_percpu_wq]
ffpu      788905  179  3.3 7285972 544824 ?      SLl  18:52 157:15 ffmpeg -hide_banner -nostats -v level+error -readrate
root      789044  0.0  0.0      0     0 ?        S    18:52   0:00 [UVM GPU1 BH]
root      789045  0.0  0.0      0     0 ?        S    18:52   0:00 [UVM GPU1 KC]
root      789086  0.0  0.0      0     0 ?        I    19:42   0:00 [kworker/u40:0-events_unbound]
root      789103  0.0  0.0      0     0 ?        I    20:10   0:00 [kworker/u40:2-ext4-rsv-conversion]
root      789112  0.0  0.0      0     0 ?        I    20:18   0:00 [kworker/u40:1-events_unbound]
root      789113  0.4  0.0  17264 10768 ?        Ss   20:20   0:00 sshd: luiz [priv]
luiz      789116  1.6  0.0  18984 10728 ?        Ss   20:20   0:00 /lib/systemd/systemd --user
luiz      789117  0.0  0.0 169680  4364 ?        S    20:20   0:00 (sd-pam)
root      789131  0.0  0.0      0     0 ?        I    20:20   0:00 [kworker/6:1]
luiz      789137  0.0  0.0  17524  6544 ?        S    20:20   0:00 sshd: luiz@pts/0
luiz      789138  0.0  0.0   7196  3804 pts/0    Ss   20:20   0:00 -bash
luiz      789141  0.0  0.0  11192  4344 pts/0    R+   20:20   0:00 ps aux
``` 

running

```
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root           1  0.0  0.0 168940 13244 ?        Ss   abr03   0:04 /sbin/init
root           2  0.0  0.0      0     0 ?        S    abr03   0:00 [kthreadd]
root           3  0.0  0.0      0     0 ?        I<   abr03   0:00 [rcu_gp]
root           4  0.0  0.0      0     0 ?        I<   abr03   0:00 [rcu_par_gp]
root           5  0.0  0.0      0     0 ?        I<   abr03   0:00 [slub_flushwq]
root           6  0.0  0.0      0     0 ?        I<   abr03   0:00 [netns]
root           8  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/0:0H-events_highpri]
root          10  0.0  0.0      0     0 ?        I<   abr03   0:00 [mm_percpu_wq]
root          11  0.0  0.0      0     0 ?        I    abr03   0:00 [rcu_tasks_kthread]
root          12  0.0  0.0      0     0 ?        I    abr03   0:00 [rcu_tasks_rude_kthread]
root          13  0.0  0.0      0     0 ?        I    abr03   0:00 [rcu_tasks_trace_kthread]
root          14  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/0]
root          15  0.0  0.0      0     0 ?        I    abr03   0:29 [rcu_preempt]
root          16  0.0  0.0      0     0 ?        S    abr03   0:05 [migration/0]
root          18  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/0]
root          19  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/1]
root          20  0.0  0.0      0     0 ?        S    abr03   0:33 [migration/1]
root          21  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/1]
root          23  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/1:0H-events_highpri]
root          24  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/2]
root          25  0.0  0.0      0     0 ?        S    abr03   0:03 [migration/2]
root          26  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/2]
root          28  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/2:0H-events_highpri]
root          29  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/3]
root          30  0.0  0.0      0     0 ?        S    abr03   0:32 [migration/3]
root          31  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/3]
root          33  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/3:0H-events_highpri]
root          34  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/4]
root          35  0.0  0.0      0     0 ?        S    abr03   0:03 [migration/4]
root          36  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/4]
root          38  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/4:0H-events_highpri]
root          39  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/5]
root          40  0.0  0.0      0     0 ?        S    abr03   0:36 [migration/5]
root          41  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/5]
root          43  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/5:0H-events_highpri]
root          44  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/6]
root          45  0.0  0.0      0     0 ?        S    abr03   0:05 [migration/6]
root          46  0.0  0.0      0     0 ?        S    abr03   0:03 [ksoftirqd/6]
root          48  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/6:0H-events_highpri]
root          49  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/7]
root          50  0.0  0.0      0     0 ?        S    abr03   0:53 [migration/7]
root          51  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/7]
root          53  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/7:0H-events_highpri]
root          54  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/8]
root          55  0.0  0.0      0     0 ?        S    abr03   0:01 [migration/8]
root          56  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/8]
root          58  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/8:0H-events_highpri]
root          59  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/9]
root          60  0.0  0.0      0     0 ?        S    abr03   0:56 [migration/9]
root          61  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/9]
root          63  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/9:0H-events_highpri]
root          64  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/10]
root          65  0.0  0.0      0     0 ?        S    abr03   0:11 [migration/10]
root          66  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/10]
root          68  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/10:0H-events_highpri]
root          69  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/11]
root          70  0.0  0.0      0     0 ?        S    abr03   1:48 [migration/11]
root          71  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/11]
root          73  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/11:0H-events_highpri]
root          74  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/12]
root          75  0.0  0.0      0     0 ?        S    abr03   0:01 [migration/12]
root          76  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/12]
root          78  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/12:0H-events_highpri]
root          79  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/13]
root          80  0.0  0.0      0     0 ?        S    abr03   0:58 [migration/13]
root          81  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/13]
root          83  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/13:0H-events_highpri]
root          84  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/14]
root          85  0.0  0.0      0     0 ?        S    abr03   0:11 [migration/14]
root          86  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/14]
root          88  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/14:0H-events_highpri]
root          89  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/15]
root          90  0.0  0.0      0     0 ?        S    abr03   5:47 [migration/15]
root          91  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/15]
root          93  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/15:0H-events_highpri]
root          94  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/16]
root          95  0.0  0.0      0     0 ?        S    abr03   0:21 [migration/16]
root          96  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/16]
root          97  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/16:0-mm_percpu_wq]
root          98  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/16:0H-events_highpri]
root          99  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/17]
root         100  0.0  0.0      0     0 ?        S    abr03   0:29 [migration/17]
root         101  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/17]
root         103  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/17:0H-events_highpri]
root         104  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/18]
root         105  0.0  0.0      0     0 ?        S    abr03   1:15 [migration/18]
root         106  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/18]
root         107  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/18:0-mm_percpu_wq]
root         108  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/18:0H-events_highpri]
root         109  0.0  0.0      0     0 ?        S    abr03   0:00 [cpuhp/19]
root         110  0.0  0.0      0     0 ?        S    abr03   2:53 [migration/19]
root         111  0.0  0.0      0     0 ?        S    abr03   0:00 [ksoftirqd/19]
root         113  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/19:0H-events_highpri]
root         128  0.0  0.0      0     0 ?        S    abr03   0:00 [kdevtmpfs]
root         129  0.0  0.0      0     0 ?        I<   abr03   0:00 [inet_frag_wq]
root         130  0.0  0.0      0     0 ?        S    abr03   0:00 [kauditd]
root         133  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/1:2-cgroup_destroy]
root         134  0.0  0.0      0     0 ?        S    abr03   0:00 [khungtaskd]
root         135  0.0  0.0      0     0 ?        S    abr03   0:00 [oom_reaper]
root         136  0.0  0.0      0     0 ?        I<   abr03   0:00 [writeback]
root         137  0.0  0.0      0     0 ?        S    abr03   0:50 [kcompactd0]
root         138  0.0  0.0      0     0 ?        SN   abr03   0:00 [ksmd]
root         141  0.0  0.0      0     0 ?        SN   abr03   0:39 [khugepaged]
root         142  0.0  0.0      0     0 ?        I<   abr03   0:00 [kintegrityd]
root         143  0.0  0.0      0     0 ?        I<   abr03   0:00 [kblockd]
root         144  0.0  0.0      0     0 ?        I<   abr03   0:00 [blkcg_punt_bio]
root         145  0.0  0.0      0     0 ?        I    abr03   0:07 [kworker/3:1-rcu_gp]
root         146  0.0  0.0      0     0 ?        I<   abr03   0:00 [tpm_dev_wq]
root         147  0.0  0.0      0     0 ?        I<   abr03   0:00 [edac-poller]
root         148  0.0  0.0      0     0 ?        I<   abr03   0:00 [devfreq_wq]
root         150  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/3:1H-kblockd]
root         151  0.0  0.0      0     0 ?        S    abr03   0:02 [kswapd0]
root         158  0.0  0.0      0     0 ?        I<   abr03   0:00 [kthrotld]
root         160  0.0  0.0      0     0 ?        I<   abr03   0:00 [acpi_thermal_pm]
root         161  0.0  0.0      0     0 ?        S    abr03   0:05 [hwrng]
root         162  0.0  0.0      0     0 ?        I<   abr03   0:00 [hfi-updates]
root         163  0.0  0.0      0     0 ?        I<   abr03   0:00 [mld]
root         164  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/2:1H-kblockd]
root         165  0.0  0.0      0     0 ?        I<   abr03   0:00 [ipv6_addrconf]
root         170  0.0  0.0      0     0 ?        I<   abr03   0:00 [kstrp]
root         174  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/11:1-mm_percpu_wq]
root         176  0.0  0.0      0     0 ?        I<   abr03   0:00 [zswap-shrink]
root         177  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/u41:0]
root         196  0.0  0.0      0     0 ?        I    abr03   0:01 [kworker/5:1-mm_percpu_wq]
root         214  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/9:1-mm_percpu_wq]
root         226  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/12:1H-kblockd]
root         238  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/8:1H-kblockd]
root         239  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/6:1H-kblockd]
root         240  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/4:1H-kblockd]
root         241  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/15:1H-kblockd]
root         243  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/14:1H-kblockd]
root         246  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/17:1H-kblockd]
root         247  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/10:1H-kblockd]
root         253  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/7:1H-kblockd]
root         281  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/11:1H-kblockd]
root         294  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/1:1H-kblockd]
root         295  0.0  0.0      0     0 ?        I<   abr03   0:03 [kworker/9:1H-kblockd]
root         296  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/5:1H-kblockd]
root         297  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/13:1H-kblockd]
root         298  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/0:1H-kblockd]
root         299  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/16:1H-kblockd]
root         300  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/18:1H-kblockd]
root         301  0.0  0.0      0     0 ?        I<   abr03   0:00 [kworker/19:1H-kblockd]
root         306  0.0  0.0      0     0 ?        I<   abr03   0:00 [ata_sff]
root         307  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_0]
root         308  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_0]
root         309  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_1]
root         310  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_1]
root         311  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_2]
root         312  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_2]
root         313  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_3]
root         314  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_3]
root         315  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_4]
root         316  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_4]
root         317  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_5]
root         318  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_5]
root         319  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_6]
root         320  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_6]
root         321  0.0  0.0      0     0 ?        S    abr03   0:00 [scsi_eh_7]
root         322  0.0  0.0      0     0 ?        I<   abr03   0:00 [scsi_tmf_7]
root         324  0.0  0.0      0     0 ?        I    abr03   0:00 [kworker/11:2-pm]
root         329  0.0  0.0      0     0 ?        I    abr03   0:02 [kworker/13:1-mm_percpu_wq]
root         330  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/16:1-mm_percpu_wq]
root         332  0.0  0.0      0     0 ?        I    abr03   0:05 [kworker/19:1-mm_percpu_wq]
root         333  0.0  0.0      0     0 ?        I    abr03   0:03 [kworker/18:1-mm_percpu_wq]
root         370  0.0  0.0      0     0 ?        S    abr03   0:13 [jbd2/sda2-8]
root         371  0.0  0.0      0     0 ?        I<   abr03   0:00 [ext4-rsv-conver]
root         410  0.0  0.3 114852 64428 ?        Ss   abr03   2:34 /lib/systemd/systemd-journald
root         443  0.0  0.0  26932  5968 ?        Ss   abr03   0:00 /lib/systemd/systemd-udevd
root         502  0.0  0.0      0     0 ?        S    abr03   0:00 [nv_queue]
root         503  0.0  0.0      0     0 ?        S    abr03   0:00 [nv_queue]
root         524  0.0  0.0      0     0 ?        S    abr03   0:00 [watchdogd]
root         527  0.0  0.0      0     0 ?        S    abr03   0:00 [irq/126-mei_me]
root         528  0.0  0.0      0     0 ?        S    abr03   2:51 [nvidia-modeset/kthread_q]
root         529  0.0  0.0      0     0 ?        S    abr03   0:00 [nvidia-modeset/deferred_close_kthread_q]
root         532  0.0  0.0      0     0 ?        I<   abr03   0:00 [cryptd]
root         623  0.0  0.0      0     0 ?        S    abr03   0:00 [jbd2/sdb-8]
root         624  0.0  0.0      0     0 ?        I<   abr03   0:00 [ext4-rsv-conver]
root         649  0.0  0.0   6608  2576 ?        Ss   abr03   0:00 /usr/sbin/cron -f
message+     652  0.0  0.0   8044  4300 ?        Ss   abr03   0:00 /usr/bin/dbus-daemon --system --address=systemd: --no
root         663  0.0  0.0  49628  7780 ?        Ss   abr03   0:00 /lib/systemd/systemd-logind
nvpd         668  0.0  0.0   6560  2008 ?        Ss   abr03   0:00 /usr/bin/nvidia-persistenced --user nvpd
root         690  0.1  0.0      0     0 ?        S    abr03   7:04 [irq/128-nvidia]
root         691  0.0  0.0      0     0 ?        S    abr03   0:00 [nvidia]
root         692  0.0  0.0      0     0 ?        S    abr03   0:56 [nv_queue]
root         693  0.0  0.0   5872   980 tty1     Ss+  abr03   0:00 /sbin/agetty -o -p -- \u --noclear - linux
root         709  0.0  0.0  15412  9188 ?        Ss   abr03   0:00 sshd: /usr/sbin/sshd -D [listener] 0 of 10-100 startu
root         742  0.0  0.0      0     0 ?        I    abr03   0:02 [kworker/17:2-mm_percpu_wq]
root        1838  0.0  0.0      0     0 ?        S    abr03   0:00 [UVM global queue]
root        1839  0.0  0.0      0     0 ?        S    abr03   0:00 [UVM deferred release queue]
root        1840  0.0  0.0      0     0 ?        S    abr03   0:00 [UVM Tools Event Queue]
root      194903  0.0  0.0      0     0 ?        I    abr03   0:01 [kworker/1:1-mm_percpu_wq]
root      241634  0.0  0.0      0     0 ?        I    abr04   0:02 [kworker/12:0-rcu_par_gp]
root      348475  0.0  0.0      0     0 ?        I    abr04   0:00 [kworker/19:2]
root      348485  0.0  0.0      0     0 ?        I    abr04   0:03 [kworker/10:2-cgroup_destroy]
root      366218  0.0  0.0      0     0 ?        I    abr04   0:00 [kworker/7:0-mm_percpu_wq]
root      371115  0.0  0.0      0     0 ?        I    abr04   0:03 [kworker/15:2-mm_percpu_wq]
root      375139  0.0  0.0      0     0 ?        I    abr05   0:00 [kworker/13:2-rcu_par_gp]
root      452990  0.0  0.0      0     0 ?        I    abr05   0:07 [kworker/3:2-events_freezable]
root      452991  0.0  0.0      0     0 ?        I    abr05   0:07 [kworker/0:1-cgroup_destroy]
root      463027  0.0  0.0      0     0 ?        I    abr05   0:00 [kworker/15:0]
ffpu      488824  0.6  0.2 186068 44900 ?        Ssl  abr05  18:28 /usr/bin/ffpapi -l 0.0.0.0:8787
root      579556  0.0  0.0      0     0 ?        I    abr05   0:00 [kworker/17:0]
root      580240  0.0  0.0      0     0 ?        I    abr05   0:01 [kworker/14:1-rcu_par_gp]
root      624324  0.0  0.0      0     0 ?        I    abr06   0:02 [kworker/12:1-cgroup_destroy]
root      624381  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/9:0]
root      631378  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/5:0]
root      665516  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/4:0-rcu_gp]
root      665517  0.0  0.0      0     0 ?        I    abr06   0:01 [kworker/2:1-mm_percpu_wq]
root      693607  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/7:2-cgroup_destroy]
root      693610  0.0  0.0      0     0 ?        I    abr06   0:00 [kworker/0:0-rcu_par_gp]
root      723213  0.0  0.0      0     0 ?        I    abr06   0:04 [kworker/6:0-mm_percpu_wq]
root      747056  0.0  0.0      0     0 ?        I    00:00   0:00 [kworker/14:2-mm_percpu_wq]
root      747057  0.0  0.0      0     0 ?        I    00:00   0:01 [kworker/8:0-mm_percpu_wq]
root      748462  0.0  0.0      0     0 ?        I    01:53   0:00 [kworker/2:2]
root      750242  0.0  0.0      0     0 ?        I    03:10   0:00 [kworker/8:1-cgroup_destroy]
root      753225  0.0  0.0      0     0 ?        I    06:16   0:00 [kworker/6:2-cgroup_destroy]
root      787720  0.0  0.0      0     0 ?        I    17:43   0:00 [kworker/4:1-mm_percpu_wq]
root      787721  0.0  0.0      0     0 ?        I    17:43   0:00 [kworker/10:1-mm_percpu_wq]
root      789086  0.0  0.0      0     0 ?        I    19:42   0:00 [kworker/u40:0-ext4-rsv-conversion]
root      789103  0.0  0.0      0     0 ?        I    20:10   0:00 [kworker/u40:2-writeback]
root      789112  0.0  0.0      0     0 ?        I    20:18   0:00 [kworker/u40:1-events_unbound]
root      789131  0.0  0.0      0     0 ?        I    20:20   0:00 [kworker/6:1-events]
ffpu      789154  0.1  0.0  28808 10332 ?        Ssl  20:23   0:00 /usr/bin/ffplayout
root      789155  0.0  0.0      0     0 ?        I    20:23   0:00 [kworker/12:2-mm_percpu_wq]
root      789158  0.0  0.0      0     0 ?        I    20:23   0:00 [kworker/8:2-cgroup_destroy]
ffpu      789169 11.2  2.2 7164844 364180 ?      SLl  20:23   0:21 ffmpeg -hide_banner -nostats -v level+info -f live_fl
root      790921  0.0  0.0      0     0 ?        I    20:23   0:00 [kworker/u40:3-flush-8:16]
root      790922  0.0  0.0      0     0 ?        I    20:23   0:00 [kworker/u40:4]
root      812010  0.0  0.0      0     0 ?        I    20:25   0:00 [kworker/10:0-mm_percpu_wq]
root      812012  0.0  0.0      0     0 ?        I    20:25   0:00 [kworker/0:2-mm_percpu_wq]
root      812013  0.0  0.0      0     0 ?        I    20:25   0:00 [kworker/0:3-cgroup_destroy]
root      812137  0.0  0.0  17388 10716 ?        Ss   20:25   0:00 sshd: luiz [priv]
luiz      812140  0.0  0.0  18984 10692 ?        Ss   20:25   0:00 /lib/systemd/systemd --user
luiz      812141  0.0  0.0 169680  4364 ?        S    20:25   0:00 (sd-pam)
luiz      812160  0.0  0.0  17648  6532 ?        S    20:25   0:00 sshd: luiz@pts/0
luiz      812161  0.0  0.0   7196  3804 pts/0    Ss   20:25   0:00 -bash
ffpu      812389  167  3.2 7276568 526844 ?      SLl  20:26   1:55 ffmpeg -hide_banner -nostats -v level+error -readrate
root      812528  0.0  0.0      0     0 ?        S    20:26   0:00 [UVM GPU1 BH]
root      812529  0.0  0.0      0     0 ?        S    20:26   0:00 [UVM GPU1 KC]
luiz      812679  0.0  0.0  11320  4896 pts/0    R+   20:27   0:00 ps aux
```

**Comment by jb-alvarado** [comment]
Wow, this bug exists since July last year...
